torch/csrc/jit/tracer.h

```diff
@@ -208,6 +204,7 @@ void addInputs(Node *n, const char * name, const at::Scalar& value);
 void addInputs(Node *n, const char * name, const at::Tensor& value);
 void addInputs(Node *n, const char * name, at::IntList value);
 void addInputs(Node *n, const char * name, at::TensorList value);
+void addInputs(Node *n, const char * name, const ArrayRef<double>& value);
```

Comment:
When do ArrayRefs of doubles appear?

Reply:
Because we support them in the JIT/Script (`DoubleList`), so when not tracing double list would be fine, but when tracing we have to error